### ISSUE TITLE
question: can we use custom templates to serve the docs?

### ISSUE THREAD

**Issue by magicalbanana** [golang/gddo#400]
Can we use custom templates to serve the docs? If so, how is that achieved?

**Comment by shurcooL** [comment]
I'm not quite sure what you mean by custom templates, can you give an example of what it'd be like? I'm guessing the answer is there is no such thing on godoc.org.

Are you already familiar with all the features that Go allows you to use in formatting the docs? https://godoc.org/github.com/natefinch/godocgo is a good explanation and demonstration of most features.

**Comment by magicalbanana** [comment]
I mean can we add styling to the html pages that get generated? 
Something like this I guess, http://dhconnelly.github.io/docgo/docgo.html.

**Comment by shurcooL** [comment]
docgo looks really cool.

> I mean can we add styling to the html pages that get generated? 

As far as I know, and assuming I understand correctly, I don't think that the list of features that godoc.org supports includes that (unless you modify its source, of course).

**Comment by magicalbanana** [comment]
@shurcooL yeah I know right? I thought this was possible in the earlier versions of gddo with a `-template` flag. Bit I could be wrong. I mean I love godocs and all but I want to add a bit of custom design to it while still be up to date with this repo's master.

**Comment by ivansantosz** [comment]
Any finding on that?

**Comment by jackwilsdon** [comment]
Looks like you can pass a custom path to the assets folder using `-assets path`.
